appveyor.yml

```diff
@@ -13,7 +13,7 @@ install:
   - git submodule update --init --recursive
 
 environment:
-  app_version: "1.2.0"
+  app_version: "1.2"
```

Comment:
3rd version number is appended as %APPVEYOUR_BUILD_VERSION%, e.g. 1.2.8 for this PR

Reply:
Cool.